### ISSUE TITLE
Periodic jobs by cron

### ISSUE THREAD

**Issue by Koc** [schmittjoh/JMSJobQueueBundle#2]
What about periodic jobs? It would be nice supports something like this

``` xml
<service id="vendorname.newsfeed.send_greetings" synthetic="true">
    <tag name="task" command="vendorname:newsfeed:send-greetings -e prod" cron-expr="10 3 * * *" />
</service>
```

**Comment by rvanlaak** [comment]
This one is closed via #16 if I'm correct?

**Comment by schmittjoh** [comment]
I think it's a separate request, but yeah we do have scheduling jobs now:
https://github.com/schmittjoh/JMSJobQueueBundle/blob/master/Resources/doc/scheduled_jobs.rst
